modules/fedimint-wallet-client/src/pegin_monitor.rs

```diff
@@ -421,7 +428,11 @@ async fn claim_peg_in(
 
         let amount = pegin_proof.tx_output().value.into();
 
-        let wallet_input = WalletInput::new_v0(pegin_proof);
+        let wallet_input = if consensus_version >= ModuleConsensusVersion::new(2, 2) {
```

Comment:
Capturing conversations out-of-band: the upgrade tests detected an edge case. Relying only on the module consensus version doesn't guarantee that the federation is aware of the UTXO. More context and fix in https://github.com/fedimint/fedimint/pull/6687